scripts/single_cursor_launcher.sh

```
@@ -0,0 +1,216 @@
+    
+    # Remove corrupted virtual environment
+    echo "🗑️  Removing corrupted virtual environment..."
+    rm -rf venv
+    
+    # Create new virtual environment with system Python
+    echo "🏗️  Creating new virtual environment..."
+    /usr/bin/python3 -m venv venv
+    
+    # Fix symbolic links
+    echo "🔗 Fixing symbolic links..."
+    cd venv/bin
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)